### PR TITLE
Add metal3 component text

### PR DESCRIPTION
**Pull request by hardys**
As a first step this is copied from the quickstart introduction

### CODE REVIEW

File: asciidoc/components/metal3.adoc [hunk 1]
```diff
@@ -10,4 +10,11 @@ ifdef::env-github[]
...
 
-Coming soon
+Metal^3^ is a https://metal3.io/[CNCF project] which provides baremetal infrastructure
+management capabilites for Kubernetes.
+
+Metal^3^ provides Kubernetes-native resources to manage the lifecycle of baremetal servers
+which support management via out-of-band protocols such as https://www.dmtf.org/standards/redfish[Redfish]
+
+It also has mature support for https://cluster-api.sigs.k8s.io/[Cluster API (CAPI)] which enables management
+of infrastructure resources accross multiple infrastructure providers via a broadly adopted vendor-neutral APIs.
```
Comment (atanasdinov) [inline asciidoc/components/metal3.adoc:20]:
```suggestion
of infrastructure resources across multiple infrastructure providers via broadly adopted vendor-neutral APIs.
```

File: asciidoc/components/metal3.adoc [hunk 2]
```diff
@@ -10,4 +10,11 @@ ifdef::env-github[]
 :warning-caption: :warning:
 endif::[]
 
-Coming soon
+Metal^3^ is a https://metal3.io/[CNCF project] which provides baremetal infrastructure
+management capabilites for Kubernetes.
+
+Metal^3^ provides Kubernetes-native resources to manage the lifecycle of baremetal servers
+which support management via out-of-band protocols such as https://www.dmtf.org/standards/redfish[Redfish]
```
Comment (atanasdinov) [inline asciidoc/components/metal3.adoc:17]:
```suggestion
which support management via out-of-band protocols such as https://www.dmtf.org/standards/redfish[Redfish].
```

**Review by atanasdinov — APPROVED** [review]

File: asciidoc/components/metal3.adoc (hunk 1 above)
Comment (hardys) [reply asciidoc/components/metal3.adoc]: Thanks, added!

File: asciidoc/components/metal3.adoc (hunk 2 above)
Comment (hardys) [reply asciidoc/components/metal3.adoc]: Thanks, added!

**Review by agracey — APPROVED** [review]
LGTM, my only note is that a link to the quickstart would likely be helpful to readers.

**Pull request merged**